composer.json

```diff
@@ -19,6 +19,10 @@
         "symfony/css-selector":  "~2.0"
     },
 
+    "require-dev": {
+        "symfony/phpunit-bridge": "~2.7"
```

Comment:
note that we should also add this dev dependency in our drivers, to make their own test fail when they use deprecated APIs (be it in Mink APIs or in Symfony ones btw). We don't want them to use deprecated APIs.